python/packages/sdk/sdk/sdk/base.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import annotations
```

Comment:
Why do we end woth `sdk/sdk/sdk/base.py` path  (`sdk` nested three times)? Is this some Python specificity?

Reply:
The first `sdk` is the bare folder that holds the package, it can be named anything, but the spec from SC-220 said to keep it as `python/packages/sdk`. The second `sdk` is the module name. This name will need to be changed for publication, as [`setuptools` demands that module names are the same as package names](https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-package-dir). 

In SC-259, I've changed the second `sdk` name, the module name, to `serverless_sdk` to match the package name that will appear on PyPI as per the `setuptools` docs, so the new full path for the packaged module will be `console/python/packages/sdk/serverless_sdk`.

The final `sdk` directory is a submodule that holds the logic for the `serverlessSdk` object itself. After conforming to the naming conventions required for publication, the final path will look like `python/packages/sdk/serverless_sdk/sdk`, where the final `sdk`submodule is aptly named for what will end up being distributed simply as `serverless_sdk/sdk` without any other nested `sdk` directories.

Reply:
Great thanks for explanations. Makes sense